73.md

```diff
@@ -0,0 +1,149 @@
+NIP-73
+======
+
+Spreadsheets
+------------
+
+`draft` `optional`
+
+This NIP provides a simple way to save spreedsheets on Nostr. 
+
+Event kind `35337` describes a workbook with `data` tags that contain the value of each cell as well as optional styling. 
+
+```js
+{
+  "kind": 35337,
+  "tags": [
+    ["d", "<unique identifier>"],
+    ["title", "Name of this spreadsheet"], // public title
+    ["data", "<sheet name>", "<column letter>", "<row number>", "<value>"],
+    ["data", "<sheet name>", "<column letter>", "<row number>", "<value>", "<style>"],
+    ["style", "<TBD>"], // Need to specify all options here. 
+  ],
+  "content": "", // reserved for private content below
+  // ...
+}
+``` 
+
+Sheets MUST be rendered in the order of their tags. 
+
+As an example: 
+
+```json
+{
+  "id": "32360b52b11616ea331aacac516494e36bd4079d8908edc8f26ad1e4acab5a53",
+  "kind": 35337,
+  "tags": [
+    [ "d", "SheetStr Demo" ],
+    [ "data", "Sheet1", "J", "25", "3" ],
+    [ "data", "Sheet1", "J", "26", "5" ],
+    [ "data", "Sheet1", "J", "27", "=SUM(J25:J26)" ]
```

Comment:
are these formula's standardized? also what about custom formulas, are they out of scope for the nip?